aptos-move/framework/aptos-framework/sources/Coin.move

```diff
@@ -189,6 +223,31 @@ module AptosFramework::Coin {
         decimals: u64,
         monitor_supply: bool,
     ) {
+        let (
+            mint_cap,
+            burn_cap
+        ) = initialize_with_capabilities<CoinType>(
+            account,
+            ASCII::string(name),
+            ASCII::string(symbol),
+            decimals,
+            monitor_supply
+        );
+
+        move_to(account, mint_cap);
+        move_to(account, burn_cap);
+    }
+
+    /// Creates a new Coin with given `CoinType` and returns minting/burning capabilities.
+    /// The given signer also becomes the account hosting the information
+    /// about the coin (name, supply, etc.).
+    public fun initialize_with_capabilities<CoinType>(
```

Comment:
not for this PR, but we should think about how to remove Cap

Reply:
Yes, I'll spend more time on the improving the Coin standard next week (in addition to staking)

Reply:
I don't follow?

Reply:
Zekun is talking about adding functionalities to remove/destroy capabilities, which I think can be useful in 2 cases:

1. Capabilities can currently be copied and handed to different accounts. There might be a need to remove these.
2. There can be a Coin that has fixed supply and not mintable/burnable. So the capabilities created would need to be destroyed.